qa/integration/services/monitoring_api.rb

```diff
@@ -0,0 +1,25 @@
+require "manticore"
+require "json"
+
+# Convenience class to interact with the HTTP monitoring APIs
+class MonitoringAPI
+
+  def pipeline_stats
+    resp = Manticore.get("http://localhost:9600/_node/stats/pipeline").body
+    stats_response = JSON.parse(resp)
+    stats_response["pipeline"]
+  end
+
+  def event_stats
+    stats = pipeline_stats
+    stats["events"]
+  end
+
+  def version
+    request = @agent.get("http://localhost:9600/")
```

Comment:
We could implement a method `#get` that takes a path and does the call + the JSON parse, it could be used by `#version` and `#pipeline_stats`